Extensions/dnSpy.BamlDecompiler/Xaml/XamlUtils.cs

```diff
@@ -48,7 +48,7 @@ static class XamlUtils {
 					sb.Append(':');
 				}
 			}
-			sb.Append(name.LocalName);
+			sb.Append(name.LocalName.Replace("_x002B_", "+"));
```

Comment:
Are we sure that undoing the escaping of the `+` character is always the right thing to do? The function you modified is used for values of XML attributes (where the `+` character is allowed since the value is stored within double quotes) as well as in the values of XML elements where I am not sure if the `+` character is valid since the value is stored raw without being placed in double quotes.

Reply:
Yeah I stated that this was sketchy in my intro. I’m remove this line from the PR.